credentials/tls.go

```diff
@@ -44,10 +44,27 @@ func (t TLSInfo) AuthType() string {
 	return "tls"
 }
 
+// cipherSuiteLookup returns the string version of a TLS cipher suite ID.
+func cipherSuiteLookup(cipherSuiteID uint16) string {
+	suites := tls.CipherSuites()
+	insecureSuites := tls.InsecureCipherSuites()
+	for i := range suites {
+		if suites[i].ID == cipherSuiteID {
+			return suites[i].Name
+		}
+	}
+	for i := range insecureSuites {
+		if insecureSuites[i].ID == cipherSuiteID {
+			return suites[i].Name
+		}
+	}
+	return ""
```

Comment:
Let's do something like this instead, which will be a little nicer if it ever happens (though it should really be impossible): `return fmt.Sprintf("unknown ID: %v", cipherSuiteID)`